### DISCUSSION TITLE
How to resolve Unexpected status code 500 for POST?

### DISCUSSION THREAD

**Discussion by imrohankataria** [strimzi/strimzi-kafka-operator#9930] [Q&A]
> All of my KafkaConnectors went into Restarting Task state last week
> This morning I tried to restart the connectors using `kubectl annotate KafkaConnector $connector strimzi.io/restart=true`
> Getting following error now
` message: Failed to restart connector <connector name>. Unexpected
      status code 500 for POST request to<cluster name>-api.kafka-connect.svc:8083/connectors/<connector name>/restart?includeTasks=false&onlyFailed=false
  `

Any resolutions please? This is our production load.

**Comment by imrohankataria** [comment]
@scholzj insights?

**Reply by scholzj** [reply]
Copying from the only thing that you shared ... it looks like it `Failed to restart connector` because it got `status code 500`. 😉

You will have to provide configurations, logs etc. You should maybe also watch [this](https://youtu.be/EkjGUNELwlE) conference talk where I tried to explain why questions like this are wasting everyone's time!
